### DISCUSSION TITLE
Summarization length

### DISCUSSION THREAD

**Discussion by onosendi** [deepgram/community#738] [General help] [answered]
Is it possible to control the length of the summaries?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@onosendi  not at this time. I'm curious though, why would you need to do that?

**Reply by onosendi** [reply]
Customers requested the feature.

“Summarize The Shawshank Redemption in 50 words or less” is a much different query than “Summarize The Shawshank Redemption.”

This system is designed for phone calls, enabling agents to quickly scan and summarize the content of each call. Some summaries are way too long.
